scripts/cibuild.sh

```diff
@@ -0,0 +1,12 @@
+set -e
```

Comment:
Script should specify a shell explicitly. Otherwise, it's behavior is a bit less predictable.

```suggestion
#! /usr/bin/bash
set -e
```